src/modules/csm/state.py

```diff
@@ -26,9 +39,10 @@ def perf(self) -> float:
 @dataclass
 class State(UserDict[ValidatorIndex, AttestationsAggregate]):
     """Tracks processing state of CSM performance oracle frame"""
-
+    l_epoch: EpochNumber = field(default_factory=int)
+    r_epoch: EpochNumber = field(default_factory=int)
     data: dict[ValidatorIndex, AttestationsAggregate] = field(default_factory=dict)
-    processed_epochs: set[EpochNumber] = field(default_factory=set)
+    epochs_to_process: set[EpochNumber] = field(default_factory=set)
```

Comment:
The thing I'm concerned the most is that we have no metric to check the state as it was using `processed_epochs`.